apps/marginfi-v2-trading/src/store/tradeStore.ts

```diff
@@ -152,7 +153,6 @@ type TradeStoreState = {
 const { programId } = getConfig();
 
 const USDC_MINT = new PublicKey("EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v");
```

Comment:
Does mrgnlabs/mrgn-common have a USDC_MINT variable? Declaring it in two places already, so might be good to extract this (also in apps/marginfi-v2-trading/src/components/common/Pool/CreatePoolDialog/components/CreatePoolLoading.tsx)